viper/modules/pehash/pehasher.py

```diff
@@ -1,37 +1,16 @@
-# This file is part of Viper - https://github.com/botherder/viper
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
 # See the file 'LICENSE' for copying permission.
 
 from __future__ import division
 
-import sys
 import bz2
-import string
 import hashlib
+import pefile
+import bitstring
 
-try:
```

Comment:
I'm ok with leaving this in.. but the reason why I removed it here is that there are only two occurences where `calculate_pehash` is used: [pe.py#L938](https://github.com/Rafiot/viper/blob/py3/viper/modules/pe.py#L938) and [pe.py#L948](https://github.com/Rafiot/viper/blob/py3/viper/modules/pe.py#L948). At this point pe.py has already checked whether HAVE_PEHASH is true... (and raised the dependency message is not)